rasa/shared/importers/autoconfig.py

```diff
@@ -8,6 +8,7 @@
 from rasa.shared.exceptions import FileNotFoundException
 import rasa.shared.utils.cli
 import rasa.shared.utils.common
+from rasa.shared.utils.common import transform_collection_to_sentence
```

Comment:
We have an internal policy to not refer to functions directly but rather to refer to them by their module. How about adding a ignore comment in this case?

Reply:
Sure. I'll do it asap.